### PR TITLE
Fix titles and some descriptions

### PR DESCRIPTION
**Pull request by tas50**
Cleanup confusing wording

### CODE REVIEW

File: core/mondoo-azure-security.mql.yaml [hunk 1]
```diff
@@ -1120,13 +1120,11 @@ queries:
         identity["type"] == "SystemAssigned" || identity["principalId"] != null
       }
   - uid: mondoo-azure-security-ensure-web-app-is-using-the-latest-tls
-    title: Ensure that Web App taking advantage of latest version of tls encryption
+    title: Ensure that Web Apps use the latest version of TLS encryption
```
Comment (misterpantz) [inline core/mondoo-azure-security.mql.yaml:1123]:
```suggestion
    title: Ensure that web apps use the latest version of TLS encryption
```

**Review by misterpantz — APPROVED** [review]

File: core/mondoo-azure-security.mql.yaml [hunk 2]
```diff
@@ -1222,11 +1220,11 @@ queries:
         secrets.all( enabled == true && expires != null )
       }
   - uid: mondoo-azure-security-ensure-the-kv-is-recoverable
-    title: Ensure it is possible to recover Key Vaults
+    title: Ensure it is possible to recover key vaults
     severity: 80
     docs:
       desc: |
-        The key Vault can contains keys, secrets and certificates. Accidental unavailability of a KV can cause serious problems in any organizations. 
+        Key vault can contains keys, secrets and certificates. Accidental unavailability of a KV can cause serious problems in any organizations.
```
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1227]:
```suggestion
        Azure Key Vault can store keys, secrets, and certificates. Accidental unavailability of a Key Vault can cause serious problems in any organization. 
```

File: core/mondoo-azure-security.mql.yaml [hunk 3]
```diff
@@ -1072,7 +1072,7 @@ queries:
         }
       }
   - uid: mondoo-azure-security-ensure-register-with-ad-is-enabled-on-app-service
-    title: Ensure that App services can authenticate with Active Directory
+    title: Ensure that app services can authenticate with Active Directory
```
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1075]:
```suggestion
    title: Ensure that App Services can authenticate with Active Directory
```

File: core/mondoo-azure-security.mql.yaml (hunk 1 above)
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1123]:
```suggestion
    title: Ensure that Web Apps use the latest available version of TLS encryption
```

File: core/mondoo-azure-security.mql.yaml [hunk 4]
```diff
@@ -1212,7 +1210,7 @@ queries:
         To update using the Microsoft Azure portal:
         1. Log in to the Microsoft Azure portal at https://portal.azure.com 
         2. Navigate to **Key vaults**.
-        3. For each Key Vaults, click on "Keys" or "Secrets"
+        3. For each key vault, select "Keys" or "Secrets"
```
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1213]:
```suggestion
        3. For each Key Vault, select "Keys" or "Secrets"
```

File: core/mondoo-azure-security.mql.yaml [hunk 5]
```diff
@@ -1222,11 +1220,11 @@ queries:
         secrets.all( enabled == true && expires != null )
       }
   - uid: mondoo-azure-security-ensure-the-kv-is-recoverable
-    title: Ensure it is possible to recover Key Vaults
+    title: Ensure it is possible to recover key vaults
     severity: 80
     docs:
       desc: |
-        The key Vault can contains keys, secrets and certificates. Accidental unavailability of a KV can cause serious problems in any organizations. 
+        Azure Key Vault can store keys, secrets, and certificates. Accidental unavailability of a Key Vault can cause serious problems in any organization.
```
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1227]: The 2nd sentence here is still super confusing

File: core/mondoo-azure-security.mql.yaml [hunk 6]
```diff
@@ -1358,19 +1356,19 @@ queries:
...
-            - Click on the "subscription name" or "resource group" or "Network security group" that the Log Alert Rule will be applied to
+            - Select the "subscription name" or "resource group" or "Network security group" that the Log Alert Rule will be applied to
           5. Verify that the selection preview shows:
             - "All network security groups" or "< your network security group >""
             - "< Resource Name >" - The subscription, group, or resource you selected
-          6. Click "Done" 
-          7. Under the Condition tab, click "Add Condition" 
+          6. Select "Done" 
+          7. Under the Condition tab, select "Add Condition" 
           8. In the "Select a signal" window, under the "Signal Name" heading, click **`Create or Update Network Security Group (Microsoft.Network/networkSecurityGroups)`**
```
Comment (tas50) [inline core/mondoo-azure-security.mql.yaml:1372]:
```suggestion
          8. In the "Select a signal" window, under the "Signal Name" heading, select **`Create or Update Network Security Group (Microsoft.Network/networkSecurityGroups)`**
```

**Pull request merged**